.gitignore

```diff
@@ -11,7 +11,6 @@
 *.skel.h
 *.a
 
-src/tests/*
```

Comment:
I added this line to avoid status like this after compilation:

```sh
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        deleted:    src/tests/Vagrantfile.erb

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        src/tests/cachestat
        src/tests/dc
        src/tests/disk
        src/tests/fd
        src/tests/filesystem
        src/tests/hardirq
        src/tests/mdflush
        src/tests/mount
        src/tests/oomkill
        src/tests/process
        src/tests/shm
        src/tests/socket
        src/tests/softirq
        src/tests/swap
        src/tests/sync
        src/tests/vfs
```

Please, can you add it back? Or to find a better way to avoid this result.